gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -248,8 +252,15 @@ def run(argv=None):
   variants |= 'FilterVariants' >> filter_variants.FilterVariants(
       reference_names=known_args.reference_names)
   if variant_merger:
-    variants |= (
-        'MergeVariants' >> merge_variants.MergeVariants(variant_merger))
+    if known_args.optimize_for_large_inputs:
+      partitions = variants | 'PartitionVariants' >> beam.Partition(
```

Comment:
The main point is to increase readability as it gives a "name" and "context" for what this block of code is doing, but feel free to leave as is for now since it's pretty small :)

Reply:
Sounds good!